shared/helperutils/helpers.go

```diff
@@ -0,0 +1,19 @@
+package helperutils
```

Comment:
please change the package name, `helperutils` utility is just too broad. I think `MerkleRoot` could be under `hashutil`, your call : )